backend/Recodo.API/appsettings.json

```diff
@@ -15,5 +15,6 @@
     "Audience": "https://localhost:4200"
   },
   "ExpireTokenTimeInMin": "1440",
+  "GyazoKey": "PFkJvKG4W4T7mzyljMVkovZM1_1AycVNmb3xK_HT80Y",

```

Comment:
Isn't it an example of sensitive data? If so, it shouldn't be committed. Save it in environment variable instead and get from there